.github/ISSUE_TEMPLATE/release.md

```diff
@@ -234,6 +234,15 @@ in the infrastructure repository. All of these tests can be run in parallel.
 
 [bintray]: https://bintray.com/beta/#/materialize/apt/materialized
 
+### Open a PR on the cloud repo enabling the new version
+
+- [ ] Issue a PR to the cloud repo to allow the released version following [the instructions][].
+- [ ] After that PR has been merged, a PR suggesting a merge from `main` -> `production` will be
+  automatically created and assigned to you. Either merge it or request that someone from the
+  cloud team merge it.
```

Comment:
```suggestion
  automatically created and assigned to you. Request somebody on the 
  [@MaterializeInc/cloud-deployers](https://github.com/orgs/MaterializeInc/teams/cloud-deployers/members)
  team review it; once approved, merge the PR and it will be automatically deployed to production.
  
```